Jenkinsfile

```diff
@@ -564,7 +639,38 @@ pipeline {
             }
         }
 
-        stage('deploy') {
+        stage('Push Release Canididate Tag') {
```

Comment:
`-Canididate`
`+Candidate`

Reply:
Fixed in eb11eec9729d84912c8df6947470fbfba550f32f.

@jandubois Any final word on ~the colour of the bikeshed~ the branch name?

Reply:
> @jandubois Any final word on ~the colour of the bikeshed~ the branch name?

Either `rc` or `release` works for me, but I prefer `rc` because it includes the `candidate` part. `release` sounds like a synonym for `master`.

Reply:
Branch name switched to `rc` in 3861cee61860961cf6356262eda6ac1e116c1cea